### ISSUE TITLE
Spine events on spinemodel

### ISSUE THREAD

**Issue by Jerakin** [defold/editor2-issues#1135]
See which spine events are available in a spine scene on a model on the spinemodel in a dropdown

**Comment by matgis-king** [comment]
Is this because you want to see which events are available when writing Lua code?

**Comment by Jerakin** [comment]
Yeah exactly, only a convince sort a thing

**Comment by Jerakin** [comment]
Need more-info still? It is only to be able to see the events to sanity check that 1. The event you expect is on the spine model and 2. The event is really called what you think.

**Comment by matgis-king** [comment]
Nah, we're good! We just need to think a bit about the best way to present this info to the user.
